streaming/src/test/scala/org/apache/spark/streaming/StreamingContextSuite.scala

```diff
@@ -176,7 +177,7 @@ class StreamingContextSuite extends FunSuite with BeforeAndAfter with Timeouts w
 
   test("stop gracefully") {
     val conf = new SparkConf().setMaster(master).setAppName(appName)
-    conf.set("spark.cleaner.ttl", "3600")
+    conf.set("spark.cleaner.ttl", "3600ms")
```

Comment:
Sean - I know the units for ```spark.cleaner.ttl``` are typically seconds, however for the sake of this test, if we set it to 3600 seconds nothing will ever happen. I think in this test it's expected to be in milliseconds, or am I mistaken?